inc/form.class.php

```diff
@@ -1313,21 +1316,22 @@ public function duplicate() {
                }
 
                $new_target_ticket = new PluginFormcreatorTargetTicket();
-               $new_target_ticket->add($update_target_ticket);
-               $new_target_item_id = $new_target_ticket->getID();
-               if (!$new_target_item_id) {
+               $update_target_ticket['title'] = $update_target_ticket['name'];
+               if (!$new_target_ticket->update($update_target_ticket)) {
```

Comment:
this is not an add of the object, but an update, we change the method.  The prepareInputForUpdate() method of the object requires we set the title key